packages/common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@explorer-1/common",
-  "version": "1.0.5",
```

Comment:
Should this maybe be considered a feature, warranting a 1.1.0 version? I don't feel strongly about it, though.

Reply:
ah right, I'll adjust that. I think I labeled the PR as a major version. I'll reflect that in the version number.

Reply:
I changed the PR label and version numbers to reflect a minor version.